specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-04-01/examples/CosmosDBDatabaseAccountCreateMax.json

```diff
@@ -141,7 +144,10 @@
           "apiProperties": {
             "serverVersion": "3.2"
           },
-          "enableAnalyticalStorage": true
+          "enableAnalyticalStorage": true,
+          "cors": {
+            "allowedOrigins": "https://test"
+          }
```

Comment:
Would this not be returned in every response ? In that case shouldnt we add it in CosmosDBDatabaseAccountCreateMin example as well ?

Reply:
Yes, you are correct. Also, I think I have defined it incorrectly as it should be an array of objects. I will make those changes

Reply:
Agreed.